### ISSUE TITLE
iOS AutoFill functionality in Dark Mode it's almost impossible to read

### ISSUE THREAD

**Issue by Greenderella** [bitwarden/mobile#1390]
## Describe the Bug

If you have Dark Mode activated in an iPhone running iOS 14.4.2 with the AutoFill functionality active it's almost impossible to read the search field.

## Steps To Reproduce

1. Activate Dark Mode
2. Activate Autofill on Bitwarden
3. Open an app that needs Bitwarden to autofill their password
4. Tap the iOS Passwords button
5. Unlock the Bitwarden app
6. Tap in the Search field and start typing

## Expected Result

The text appears as white on a dark gray background. See Screenshot 2

## Actual Result

The text appears as black on a dark gray background. See Screenshot 1

## Screenshots or Videos

![Screenshot 1](https://user-images.githubusercontent.com/5418577/116540078-f9febe00-a8e9-11eb-9290-fcf8b00dbfe3.png)
![Screenshot 2](https://user-images.githubusercontent.com/5418577/116540086-fb2feb00-a8e9-11eb-81b1-97f6977a937e.png)

## Environment

- Device: iPhone SE 2020
- Operating system: iOS 14.4.2
- Build Version (go to "Settings" → "About" in the app): 2.9.1
- Is this a Beta release? N

**Comment by g-zhang** [comment]
Any update on this? I'm hitting the issue as well. The problem is even worse and a darker theme like Black. See below:

# Screenshots or Videos
https://user-images.githubusercontent.com/6549847/158691431-36a56ec1-5959-41f4-a3f3-3d0ac2e31c2e.mp4

# Environment
- Device: iPhone 13 Pro
- Operating system: iOS 15.4
- Build Version (go to "Settings" → "About" in the app): 2.16.3
- Is this a Beta release? N
